### ISSUE TITLE
touhou 13 crashes during dialogue of the last boss at stage 2 in japanese version

### ISSUE THREAD

**Issue by saufall** [thpatch/thcrap#228]
I have touhou 13 patched to the English language and applied vpatch.
but i wanted to play Japanese so I patched through the outdated thcrap_configure.exe to japanese version (patch #304).
and run the game windowed at 1280*960 resolution.
i then accidentally clicked the maximization button so it becomes fullscreen. when i changed it back to window it now only has a small window (around 680*720 or so)
so i changed the resolution in vpatch.ini to 128*960. it worked. but the game crashes at stage 2 when the last boss gives her dialogue before entering combat. 
i tried it windowed, fullscreen. i updated the thcrap and reinstalled the jpn pack through thcrap.exe. i made sure i installed only jpn patch and the english patch is not mixed with it. but it still crashes.
is it possibly due to a bug, or a corrupted game file? or something with vptach?

**Comment by brliron** [comment]
"at stage 2 when the last boss gives her dialogue before entering combat" sounds like something that would be caused by thcrap.

The patch lang_jp tend to not be really well maintained, because, well, if someone wants to play in Japanese, they don't need to apply more Japanese over their Japanese game, it's already in Japanese. I don't even know why we have this patch.

I suggest re-running the configuration tool (the outdated one or the current one, it doesn't really matter) and to select only the patch nmlgc/base_tsa.

**Comment by saufall** [comment]
Thank you.
Can I just use the vpatch as I dont need a language patch anyway? Does it run the game in the unaltered japanese verision if i simply run vpatch.exe?

**Comment by brliron** [comment]
Yes, if you don't use a thcrap patch, you don't really need thcrap. Vpatch works just fine without thcrap.
